habitat/plan.sh

```diff
@@ -9,10 +9,6 @@ pkg_license=('Apache-2.0')
 pkg_deps=(
   core/coreutils
 )
-# This is set to force the source path to the root level during automate builds
-# You must build plan from the root dir with command `build habitat`
-# Recommended to execute `scripts/hab-it`
-SRC_PATH="../"
```

Comment:
Are we ignoring our own comments now? :)

Does habitat now continue to search up the dir tree to find the Gemfile?

Reply:
I am not sure, but I wanted to roll the dice to see if the changes to the Ruby scaffolding fixed this issue.

Reply:
I think this change is a gamble when we can just look at the hab project. Also, this is testable locally. Is there a reason you are submitting a change through automate for this?

Reply:
Just fixed one thing, but works locally (now). From the comment I assumed it was only an issue when running this via automate/workflow.

Reply:
Let's pair on this and maybe it will jug our memory. That work was all done in haste.

Reply:
Just as a note, I'm 99% sure you still need this to make it work with habitat_build